src/main/java/org/schemaspy/output/html/mustache/diagrams/MustacheTableDiagramFactory.java

```diff
@@ -77,38 +87,42 @@ public MustacheTableDiagramFactory(DotFormatter dotProducer, MustacheDiagramFact
             oneDiagram.setActive(true);
             diagrams.add(oneDiagram);
 
-            WriteStats twoStats = new WriteStats(stats);
-            try (PrintWriter dotOut = Writers.newPrintWriter(twoDegreesDotFile)) {
-                impliedConstraints = dotProducer.writeRealRelationships(table, true, twoStats, dotOut, outputDir);
-            }
-
-            if (sameWritten(oneStats, twoStats)) {
-                Files.deleteIfExists(twoDegreesDotFile.toPath()); // no different than before, so don't show it
-            } else {
-                diagrams.add(mustacheDiagramFactory.generateTableDiagram("Two degrees", twoDegreesDotFile, table.getName() + ".2degrees"));
-            }
-
-            if (!impliedConstraints.isEmpty()) {
-                WriteStats oneImplied = new WriteStats(stats);
-                try (PrintWriter dotOut = Writers.newPrintWriter(oneImpliedDotFile)) {
-                    dotProducer.writeAllRelationships(table, false, oneImplied, dotOut, outputDir);
-                }
-                MustacheTableDiagram oneImpliedDiagram = mustacheDiagramFactory.generateTableDiagram("One implied", oneImpliedDotFile, table.getName() + ".implied1degrees");
-                oneImpliedDiagram.setIsImplied(true);
-                diagrams.add(oneImpliedDiagram);
-                WriteStats twoImplied = new WriteStats(stats);
-                try (PrintWriter dotOut = Writers.newPrintWriter(twoImpliedDotFile)) {
-                    dotProducer.writeAllRelationships(table, true, twoImplied, dotOut, outputDir);
-                }
-                if (sameWritten(oneImplied, twoImplied)) {
-                    Files.deleteIfExists(twoImpliedDotFile.toPath());
-                } else {
-                    MustacheTableDiagram twoImpliedDiagram = mustacheDiagramFactory.generateTableDiagram("Two implied", twoImpliedDotFile, table.getName() + ".implied2degrees");
-                    twoImpliedDiagram.setIsImplied(true);
-                    diagrams.add(twoImpliedDiagram);
-                }
-                return diagrams;
+            WriteStats oneImplied = new WriteStats(stats);
+            try (PrintWriter dotOut = Writers.newPrintWriter(oneImpliedDotFile)) {
+                dotProducer.writeAllRelationships(table, false, oneImplied, dotOut, outputDir);
             }
+            MustacheTableDiagram oneImpliedDiagram = mustacheDiagramFactory.generateTableDiagram("One implied", oneImpliedDotFile, table.getName() + ".implied1degrees");
+            oneImpliedDiagram.setIsImplied(true);
+            diagrams.add(oneImpliedDiagram);
+ 
+
+	        if (relationDegreeDepth == 2) {
```

Comment:
The ordering here will change the layout of the report.

1 degree:
1 real, 1 implied,
2 degrees:
1 real, 1 implied, 2 real, 2 implied

Can't say it's a huge issue, but it will change, grupping on degree instead of real and implied.